### PR TITLE
Update spm quickstarts with CI script

### PR DESCRIPTION
**Pull request by jrcrespoh**
- [x] abtesting
- [x] crashlytics
- [x] database
- [x] performance

### CODE REVIEW

File: scripts/test_spm.sh [hunk 1]
```diff
@@ -20,32 +20,62 @@
...
+    CODE_SIGNING_ALLOWED=NO
+    OTHER_SWIFT_FLAGS=${SWIFT_DEFINES}
+)
+
+function xcb() {
+    echo xcodebuild "$@"
+    xcodebuild "$@" | xcpretty
+}
+
+# xcodebuild -project "$PROJECT" -scheme "$SCHEME" -showdestinations
```
Comment (morganchen12) [inline scripts/test_spm.sh:72]: nit: remove this if it's unused

File: .github/workflows/crashlytics.yml [hunk 2]
```diff
@@ -53,49 +53,53 @@ jobs:
...
-#           CODE_SIGN_IDENTITY="" \
-#           CODE_SIGNING_REQUIRED=NO \
-#           CODE_SIGNING_ALLOWED=NO
-#         env:
-#           SAMPLE: Crashlytics
+          DIR: crashlytics
+          OS: watchOS
+          DEVICE: Apple Watch Series 5 - 44mm
+          SWIFT_DEFINES: ""
+          TEST: false
```
Comment (morganchen12) [inline .github/workflows/crashlytics.yml:105]: This is much nicer than the previous CI script, great work!

**Review by morganchen12 — APPROVED** [review]
LGTM, but please get Paul's approval before merging.

File: scripts/test_spm.sh [hunk 3]
```diff
@@ -20,32 +20,62 @@
...
-     -destination "platform=${OS} Simulator,name=${DEVICE}" \
-     build \
-     test \
-     ONLY_ACTIVE_ARCH=YES \
-     OTHER_SWIFT_FLAGS=${SWIFT_DEFINES} \
-     | xcpretty
+# Check Xcode version when testing watchOS
+if [[ "$TEST" == true && "$OS" == watchOS ]]; then
+    version="$(xcode-select -p)"
+    if [[ "$version" != "/Applications/Xcode_13.0.app" && \
```
Comment (paulb777) [inline scripts/test_spm.sh:26]: Please investigate a generalized test for any Xcode >= 12.5. I'd prefer for this not to start breaking with 13.1 etc.

Comment (paulb777) [reply scripts/test_spm.sh:26]: Something like https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/build.sh#L97 could help

**Review by paulb777 — COMMENTED** [review]
This is a great improvement for maintainability. Thanks!

One thing to solve and then LGTM

File: scripts/test_spm.sh (hunk 1 above)
Comment (jrcrespoh) [reply scripts/test_spm.sh:72]: Done ✅

File: scripts/test_spm.sh (hunk 3 above)
Comment (jrcrespoh) [reply scripts/test_spm.sh:26]: Done ✅

**Review by paulb777 — APPROVED** [review]

**Review by granluo — APPROVED** [review]

**Pull request merged**